integration_tests/dbt_project.yml

```diff
@@ -1,31 +1,62 @@
 name: 'shopify_holistic_reporting_integration_tests'
-version: '0.3.0'
+version: '0.4.0'
 profile: 'integration_tests'
 config-version: 2
 
+clean-targets:         # directories to be removed by `dbt clean`
+  - "target"
+  - "dbt_packages"
+  - "dbt_modules"
 
 vars:
   shopify_source:
-    customer_source: "{{ ref('shopify_customer_data') }}"
-    order_line_refund_source: "{{ ref('shopify_order_line_refund_data') }}"
-    order_line_source: "{{ ref('shopify_order_line_data') }}"
-    order_source: "{{ ref('shopify_order_data') }}"
-    product_source: "{{ ref('shopify_product_data') }}"
-    order_adjustment_source: "{{ ref('shopify_order_adjustment_data') }}"
-    product_variant_source: "{{ ref('shopify_product_variant_data') }}"
-    refund_source: "{{ ref('shopify_refund_data') }}"
-    transaction_source: "{{ ref('shopify_transaction_data') }}"
-
+    shopify_schema: 'shopify_holistic_reporting_integration_tests_1'
+    # shopify_customer_identifier:          "customer"
```

Comment:
they are indeed meant to be commented out! i can remove them if they're confusing, just figured i'd leave examples in here

IMO changes to seed files don't need to go in the CHANGELOG since they don't affect users at all...but are there reasons that users would want to be aware of these sorts of changes? 🤔